rllib/algorithms/algorithm.py

```diff
@@ -2259,6 +2259,14 @@ def _is_multi_agent(self):
                         return isinstance(self, MultiAgentEnv)
 
                 return env_id, lambda cfg: _wrapper.remote(cfg)
+            # Gymnasium.Env-subclass: Also go through our RLlib gym-creator.
+            elif issubclass(env_specifier, gym.Env):
```

Comment:
is gym.Env here actually gymnaisum.Env? That's how I interpret the comment.

Reply:
Correct. Yeah, it's a little misleading. From 2.3 on, we always mean gymnasium in our code/comments when we write `gym` as we always import gymnasium via `import gymnasium as gym`.

Reply:
Fixed the comment to be more consistent with this nomenclature.